src/MusicStore/Models/ShoppingCart.cs

```diff
@@ -78,14 +78,14 @@ public int RemoveFromCart(int id)
 
         public void EmptyCart()
```

Comment:
Nit: Any reason some methods like this one that performs queries isn't async? It seems most of what I saw above is async.